src/shutdown/shutdown.c

```diff
@@ -379,6 +378,18 @@ int main(int argc, char *argv[]) {
                                           watchdog_device);
         }
 
+        watchdog_usec = getenv("WATCHDOG_USEC");
+        if (watchdog_usec) {
+                usec_t usec;
+
+                r = parse_time(watchdog_usec, &usec, 1);
+                if (r >= 0)
+                        r = watchdog_set_timeout(&usec);
+                if (r < 0)
+                        log_warning_errno(r, "Failed to start watchdog with timeout=%s, ignoring: %m",
+                                          watchdog_usec);
```

Comment:
watchdog_set_timeout() already logs on all errors it returns errors for on LOG_WARNING level, no need to log again in that case. should be sufficient to log in the parse failure case.

Reply:
Fixed.